api/objects.proto

```diff
@@ -25,6 +26,8 @@ message Meta {
 
 // Node provides the internal node state as seen by the cluster.
 message Node {
+	option (docker.protobuf.plugin.store_object) = { table: "node" };
```

Comment:
It is not used yet. I anticipated automatically generating the index schema, but it's not there yet. On further consideration, if/when this happens it should just be generated from a lowercased version of the object name. I'll remove this.

Note that #2034 extends this protobuf option with further fields.